src/modes/summary/Summary.svelte

```diff
@@ -1,6 +1,5 @@
 <script>
   import IndicatorTable from './IndicatorTable.svelte';
-  import Overview from './Overview.svelte';
```

Comment:
Does this mean we can remove `src/modes/summary/Overview.svelte`?  Does that then mean we can remove `defaultCasesSensor` and `defaultDeathSensor` from `src/stores/index.ts`?  Perhaps even move `defaultHospitalSensor` into this file?